src/test/java/de/tum/in/www1/artemis/connector/BambooRequestMockProvider.java

```diff
@@ -224,95 +215,32 @@ else if (buildPlanToBeReturned != null) {
         }
     }
 
-    public void mockAddTrigger(String buildPlanKey, String repository) throws URISyntaxException, IOException {
```

Comment:
Why is this mock method no longer needed?

Reply:
because we don't have triggers in Bamboo anymore